src/inc/quic_datapath.h

```diff
@@ -207,7 +207,12 @@ typedef struct CXPLAT_ROUTE {
     uint8_t LocalLinkLayerAddress[6];
     uint8_t NextHopLinkLayerAddress[6];
 
-    uint16_t DatapathType; // CXPLAT_DATAPATH_TYPE
+    union {
+        struct {
+            uint16_t DatapathType: 15; // CXPLAT_DATAPATH_TYPE
+            uint8_t UseQTIP: 1;        // TRUE if the route is using QTIP
+        };
+    };
```

Comment:
You shouldn't need all the union and struct stuff.
```suggestion
    union {
        struct {
            uint16_t DatapathType: 15; // CXPLAT_DATAPATH_TYPE
            uint8_t UseQTIP: 1;        // TRUE if the route is using QTIP
        };
    };
```

Reply:
Your suggested change is the same as what I already have.